Turn/src/Game.cpp

```diff
@@ -340,10 +343,21 @@ void Game::HowToPlay() {
 			<< "Whetstone: Restores your weapon's sharpness." << endl << endl
 			<< "Good luck and have fun!" << endl << endl
 			<< "1) Quit" << endl << endl << "> ";
-		cin >> choice;
+		choice = GetChoice();
 		switch (choice) {
 		case 1:
 			MainMenu();
 		}
 	}
 }
+
+int Game::GetChoice()
+{
+	int choice = -1;cin >> choice;
+	while (!(cin >> choice)) {
+		cin.clear();
+		cin.ignore(numeric_limits<streamsize>::max(), '\n');
+		cout << "Invalid input.  Please try again: ";
```

Comment:
I would prefer calling `Sleep(SLEEP_MS)` after the error message, then redraw the main menu. This way the screen won't be littered with invalid input messages if they enter a lot of incorrect keys.